From wfp5p@virginia.edu  Mon Jul 27 14:12:23 2009
From: Bill Pemberton <wfp5p@virginia.edu>
Date: Mon, 27 Jul 2009 16:47:43 -0400
Subject: Staging: hv: remove VMBUS_MSGINFO typedef
To: greg@kroah.com
Cc: hjanssen@microsoft.com
Message-ID: <1248727665-23969-22-git-send-email-wfp5p@virginia.edu>


Note that this struct doesn't appear to be used anywhere and can
probably be removed.

Signed-off-by: Bill Pemberton <wfp5p@virginia.edu>
Cc: Hank Janssen <hjanssen@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/hv/VmbusPrivate.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/hv/VmbusPrivate.h
+++ b/drivers/staging/hv/VmbusPrivate.h
@@ -97,7 +97,7 @@ struct VMBUS_CONNECTION {
 };
 
 
-typedef struct _VMBUS_MSGINFO {
+struct VMBUS_MSGINFO {
 	/* Bookkeeping stuff */
 	LIST_ENTRY			MsgListEntry;
 
@@ -106,7 +106,7 @@ typedef struct _VMBUS_MSGINFO {
 
 	/* The message itself */
 	unsigned char		Msg[0];
-} VMBUS_MSGINFO;
+};
 
 
 
